### ISSUE TITLE
PopularStreams : NumberFormatException > ∞

### ISSUE THREAD

**Issue by HakumenNC** [adriens/imgflip4j#8]
`∞` cause a `NumberFormatException` :

![image](https://user-images.githubusercontent.com/2644417/125777948-f83a04cc-fbfc-41b6-9e3b-00cbcf4a13b9.png)

```java
Exception in thread "main" java.lang.NumberFormatException: For input string: "∞"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at com.github.adriens.imgflip.sdk.imgflip.sdk.PopularStreamCrawler.getPopularStreams(PopularStreamCrawler.java:81)
	at com.github.adriens.imgflip.sdk.imgflip.sdk.PopularStreamCrawler.getPopularStreams(PopularStreamCrawler.java:43)
	at com.github.adriens.imgflip.sdk.imgflip.sdk.PopularStreamCrawler.getPopularStreams(PopularStreamCrawler.java:34)
	at com.github.adriens.imgflip.sdk.imgflip.sdk.PopularStreamCrawler.main(PopularStreamCrawler.java:95)
```

**Comment by adriens** [comment]
Hi @HakumenNC , how did you achieve to generate this bug ? It looks like a meme byt itself !!! 🙄

**Comment by adriens** [comment]
Ok, got it, awesome, THIS IS AWESOME !!! 👏

**Comment by adriens** [comment]
https://stackoverflow.com/questions/18398690/how-should-i-parse-infinity-with-double

```java
if(yourstring.ToLower() == "infinity")
{
   yourdouble = double.PositiveInfinity;
}
```

**Comment by adriens** [comment]
As we are using an `int`, maybe should we use `Integer.MAX_VALUE` ❔

**Comment by HakumenNC** [comment]
`Integer.MAX_VALUE` sound like the better solution for me

**Comment by HakumenNC** [comment]
@adriens 

Ready 🚀 to release !

**Comment by adriens** [comment]
So you started to fix it LOL I should call you "🚀 man"

**Comment by HakumenNC** [comment]
Fixed on version [`2.1`](https://github.com/ImgFlip-Lovers/imgflip4j/releases/tag/2.1)
